node/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "moduleResolution": "node",
     "isolatedModules": true,
     "noEmit": true,
-    "strict": true
+    "strict": true,
+    "skipLibCheck": true
```

Comment:
Necessary for Vitest, unfortunately 😕. But also, this is fine to add IMO as this PR does add more typechecking to tests.